### PR TITLE
Grid loader

### PR DESCRIPTION
**Pull request by dvdfu**

### CODE REVIEW

File: public/js/game.js [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
+	{id: 2, shape: 'square', backgroundColor: 'black', shapeColor: 'red'},
+	{id: 3, shape: 'square', backgroundColor: 'black', shapeColor: 'yellow'},
+	{id: 4, shape: 'square', backgroundColor: 'grey', shapeColor: 'blue'},
+	{id: 5, shape: 'square', backgroundColor: 'grey', shapeColor: 'red'},
+	{id: 6, shape: 'square', backgroundColor: 'grey', shapeColor: 'yellow'},
+	{id: 7, shape: 'square', backgroundColor: 'white', shapeColor: 'blue'},
+	{id: 8, shape: 'square', backgroundColor: 'white', shapeColor: 'red'},
+	{id: 9, shape: 'square', backgroundColor: 'white', shapeColor: 'yellow'}
+	];
+
```
Comment (Nosajool) [inline public/js/game.js]: is this test data?

File: public/js/game.js [hunk 2]
```diff
@@ -35,4 +49,35 @@ socket.on('user setup', function(){
...
+		}
+
+		$('#row' + Math.floor(i / 3)).append(
+			'<div class="col-xs-4">' +
+			'<div class="tile color-' + bgColor + '">' +
+			'<svg class="shape-svg" viewBox="0 0 100 100" preserveAspectRatio="none">' +
+			shape +
+			'</svg>' + '</div>' + '</div>');
+	}
+}
```
Comment (Nosajool) [inline public/js/game.js]: \n

File: public/js/game.js [hunk 3]
```diff
@@ -35,4 +49,35 @@ socket.on('user setup', function(){
...
+	for (var i = 0; i < tiles.length; i++) {
+		tile = tiles[i];
+
+		if (tile.backgroundColor === 'black') {
+			bgColor = 'red-light';
+		} else if (tile.backgroundColor === 'grey') {
+			bgColor = 'blue-light';
+		} else if (tile.backgroundColor === 'white') {
+			bgColor = 'yellow-light';
+		}
```
Comment (Nosajool) [inline public/js/game.js:63]: lol....black maps to red-light, grey maps to blue-light, white maps to yellow-light

File: public/js/game.js (hunk 1 above)
Comment (dvdfu) [inline public/js/game.js]: yes

**Comment by Nosajool** [comment]
use mathew's PR for the actual tile data.

**Comment by yawhide** [comment]
:8ball:

**Pull request merged**